packages/nimble-components/src/rich-text/specs/README.md

```diff
@@ -176,6 +291,21 @@ _Events_
     2. This event will not fire when there are no changes made to the content of the editor. For example, all mouse events, selecting the texts, state
        changes, etc,
 
+-   `mention-update`: This can be achieved through Tiptap's `onUpdate()` and `onStart()` methods in `render` function in
+    [suggestion](https://tiptap.dev/api/utilities/suggestion#render) configurations. This event fires with the `eventData` containing the
+    current text that is added after the `@` character.
+
+    This event will be triggered in the following scenarios:
+
+    1. When a user inserts the character (e.g., `@`) into the editor, which activates the mention popup.
+    2. When a user adds or removes text after inserting the mention character into the editor.
+    3. When a user repositions the cursor between the text segments added after the mention character.
```

Comment:
> Do you mean that if I type "@ab" and then hit the left arrow key to put the cursor between "a" and "b", we emit an event?

As I replied in the [other comment](https://github.com/ni/nimble/pull/1594#discussion_r1361796266), it is because of the default behavior of Tiptap. Based on the decision from that comment, I will update the event accordingly here.

Reply:
@vivinkrishna-ni It will be good to cross link the Accessibility section here, as this section comes before accessibility section and some specific details of why the event is fired on reposition and what the consumers will do with it is provided there. 

You can also copy and add a brief here alternatively.

Reply:
I have cross-linked the `accessibility` section here and rephrased the sentences to explicitly mention that this event is for filtering the options in the client side.